packages/playground/src/layouts/RootLayout.tsx

```diff
@@ -9,9 +9,9 @@ const RootLayout: React.FC = () => {
   const showNavbar = !hideNavbarPaths.includes(location.pathname)
 
   return (
-    <div className="bg-background flex">
+    <div className="bg-background" style={{ display: 'flex' }}>
```

Comment:
add tw class `flex` instead of inline style

Reply:
solved to the flex tag in tailwind